### ISSUE TITLE
update documentation regarding go 1.20 coverage tools

### ISSUE THREAD

**Issue by mlctrez** [agnivade/wasmbrowsertest#51]
This issue to get your thoughts on a documentation update.  
I think it would be a good add, but understand if it seems out of scope for this project's readme.

The code coverage tools introduced in go 1.20 produce multiple [coverage data files](https://go.dev/testing/coverage/#working) in binary format, one of the drivers for #49 

Using `-test.coverprofie` still works after #49 and produces the single coverage output file.

I'd like to submit a PR to add instructions (at a high level) that read:
* When generating coverage with go 1.20 and later:
  * Prefer the `-test.gocoverdir` option instead of `-test.coverprofile` for coverage generation.
    * Having less data across the http fs api calls is the primary benefit.
  * Post process the coverage data files using the go covdata tool.
    * As in `go tool covdata -i <gocoverdir> -o coverage.out`
    * A additional benefit is that wasm and nonwasm coverage runs can be merged with this command.

Thanks,
Matt

**Comment by agnivade** [comment]
Makes sense. Please feel free to go ahead. Thanks!

**Comment by mlctrez** [comment]
Here you go: https://github.com/agnivade/wasmbrowsertest/pull/52

Link to [readme](https://github.com/mlctrez/wasmbrowsertest/tree/readme-coverage#a-tip-on-coverage-data-using-go-120-or-later) in my branch.

**Comment by agnivade** [comment]
Closing as this is resolved now.
